src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/ApplicationRepository.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.Linq;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Handles applicationMetadata repository
+    /// </summary>
+    public class ApplicationRepository : IApplicationRepository
+    {
+        private readonly Uri _databaseUri;
+        private readonly Uri _collectionUri;
+        private readonly string databaseId;
+        private readonly string collectionId = "applications";
```

Comment:
We should agree on a naming standard for the collections. The existing collection for instances starts with a capital letter and is in singular form.